Workflow/Sources/Workflow/Protocols/FlowRepresentable.swift

```diff
@@ -22,10 +22,11 @@ public protocol FlowRepresentable {
 
     var _workflowPointer: AnyFlowRepresentable? { get set }
     var _workflowUnderlyingInstance: Any { get }
-    /// instance: A method to return an instance of the `FlowRepresentable`
-    /// - Returns: `AnyFlowRepresentable`. Specifically a new instance from the static class passed to a `Workflow`
-    /// - Note: This needs to return a unique instance of your view. Whether programmatic or from the storyboard is irrelevant
-    static func instance() -> Self
```

Comment:
This should be marked as `unavailable` and point consumers to how to upgrade.

Reply:
In a frustrating and shocking twist members can only be marked unavailable in objective-c protocols and it is completely impossible to represent a FlowRepresentable in Objective-C